packages/lesswrong/client/datadogRum.ts

```diff
@@ -25,15 +25,16 @@ export async function initDatadog() {
     site: 'datadoghq.com',
     service:'eaforum-client',
     env: ddEnv,
-    // version: '1.0.0',
-    tracingSampleRate: ddTracingSampleRate.get(),
-    sampleRate: ddRumSampleRate.get(),
+    version: '2.0.0',
```

Comment:
Previously there was no version, I added one seeing as there might be a jump in some metrics due to the upgrade